backend/src/api/auth/auth.controller.js

```diff
@@ -74,3 +74,22 @@ exports.logoutUser = async (req, res, next) => {
 function removeExpiredTokens(array) {
   return array.filter(item => item.expires > Date.now());
 }
+
+exports.AuthGoogle = async (req, res, next) => {
+  const user = req.user;
+  const existingUser = await UserModel.findById(user._id);
+  const expiresIn = 2 * 24 * 60 * 60;
```

Comment:
could you move it to env variable or sort of a constant at least? This value the same for regular sign-in and google sign-in

Reply:
ok